### DISCUSSION TITLE
Closing window keeps Sunshine Open

### DISCUSSION THREAD

**Discussion by OUSooner911** [LizardByte/community#126] [Sunshine] [answered]
I wish when I pressed the X in the window of sunshine, it minimized to the system tray and ran in the background. If I wanted to fully close it, then I could right-click the icon and press a new menu item called "exit"

**Comment by Senorsen** [comment]
If in Windows OS, try start Sunshine using the service `sunshinesvc` instead of the shortcut with "foreground".
![image](https://user-images.githubusercontent.com/4176091/235189173-01f2b2c2-32e4-4513-9483-77f0a1cf2d25.png)

**Comment by ReenigneArcher** [comment, marked as answer]
Recommended option is to run Sunshine as a service. There is no Sunshine window when running as a service.

Additionally, the system tray icon has both restart and exit options.
